components/molecule/autosuggest/src/index.js

```diff
@@ -34,23 +34,24 @@ const getIsTypeableKey = key => {
   return key.length === 1 || keysEdit.includes(key)
 }
 
-const MoleculeAutosuggest = ({multiselection, ...props}) => {
-  const {
-    refMoleculeAutosuggest: refMoleculeAutosuggestFromProps,
-    children,
-    onToggle,
-    onChange,
-    onBlur,
-    onEnter,
-    onFocus,
-    isOpen,
-    keysCloseList,
-    keysSelection,
-    disabled,
-    errorState,
-    state
-  } = props
-
+const MoleculeAutosuggest = ({
+  multiselection,
```

Comment:
I would suggest to order props to make code more readable. Here and during this PR, we could do it. It's a suggestion, feel free!